javascript/javascript3/week1/WeatherApp/main.js

```diff
@@ -0,0 +1,130 @@
+const btn1 = document.getElementById('btnGetCity');
+const btn2 = document.getElementById('btnGetLoc');
+const displayMessage = document.getElementById("displayMessage");
+const displayWeather = document.getElementById("displayWeather");
+
+//onload show weather for saved city or location in localstorage
+window.onload = () => {
+    if (localStorage.getItem("city")) {
+        console.log(localStorage.getItem("city"));
```

Comment:
Since you use the stored located a few different places, you could save to a const instead
```suggestion
window.onload = () => {
    const localCity = localStorage.getItem("city")
    if (localCity) {
        console.log(localCity);
```